application/config.json.template

```diff
@@ -44,5 +44,9 @@
            "Other"
        ],
        "categoryRoleSuffix": " - Helper"
-   }
+   },
+   "blacklistedFileExtension":[
+    "exe",
+    "tar"
```

Comment:
fix indentation

Reply:
i think this is the right indentation?

```JSON
  "blacklistedFileExtension": [
    "exe",
    "tar"
  ]
```
or should it be?:

```JSON
  "blacklistedFileExtension": [
      "exe",
      "tar"
  ]
```

Reply:
The file uses 4 spaces for indents